blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -227,6 +327,26 @@ exports.fetchLogsInjectStart = function fetchLogsInjectStart(url, start, onSucce
             }
         });
 };
+
+/**
+ * Determines if the provided object is a "run" type
+ */
+function isRun(obj) {
+    return obj._class === 'io.jenkins.blueocean.rest.impl.pipeline.PipelineRunImpl'
```

Comment:
@kzantow from vivek: 

"Scan blueocean-rest-impl and blueocean-rest module for @capabilities annotation on the relevant Java classes that corresponds to relevant json object. Basically knowledge of backend model. If ui folks can tell what generic capabilities they are looking for I can help them create a doc with corresponding capabilities name. 

Few known things come to my mind are, run, branch, types of runs, test types, log... 
"

He is on the road this evening, but is considering adding common ones to README if that will help.